website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md

```diff
@@ -0,0 +1,160 @@
+---
+description: Follow the steps to upgrade the Appsmith community Kubernetes installation to a Business Edition.
+---
+
+# Kubernetes
```

Comment:
@danciaclara and @sharat87 - this file is ready for review. Please review and add your comments.